### DISCUSSION TITLE
unable to access server

### DISCUSSION THREAD

**Discussion by Kushwaha2021** [saran-surya/email_auth#29] [Q&A] [answered]
-- package Error ---
I/flutter (11094): email-auth >> The remote server is not a valid.
I/flutter (11094): email-auth >> configured server : "https://dashboard.heroku.com"

**Comment by saran-surya** [comment, marked as answer]
Hi @Kushwaha2021 I this that dashboard is not a server i believe. Your configured server will have XXX.herokuapp.com,
Kindly go though the complete steps here.

https://saran-surya.github.io/email-auth-node/
